module/plugins/hooks/Captcha9Kw.py

```diff
@@ -1,254 +1,309 @@
+                                    cpm_option = hosteroption[1]
+                            elif hosteroption[0].lower() == 'selfsolve' and hosteroption[1].isdigit():
+                                    selfsolve_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'confirm' and hosteroption[1].isdigit():
+                                    confirm_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'max' and hosteroption[1].isdigit():
+                                    max_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'min' and hosteroption[1].isdigit():
+                                    min_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'case-sensitive' and hosteroption[1].isdigit():
+                                    case_sensitive_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'math' and hosteroption[1].isdigit():
+                                    math_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'numeric' and hosteroption[1].isdigit():
+                                    numeric_option = hosteroptions[1]
+                            elif hosteroption[0].lower() == 'phrase' and hosteroption[1].isdigit():
+                                    phrase_option = hosteroptions[1]
+
+        for _ in xrange(1, 5, 1):
             try:
-                res = self.load(self.API_URL, post=post_data)
+                response = getURL(https + self.API_URL, post = {
+                    "apikey": self.config.get("passkey"),
+                    "prio": prio_option,
+                    "confirm": confirm_option,
+                    "maxtimeout": timeout_option,
+                    "selfsolve": selfsolve_option,
+                    "captchaperhour": cph_option,
+                    "captchapermin": cpm_option,
+                    "case-sensitive": case_sensitive_option,
+                    "min_len": min_option,
+                    "max_len": max_option,
+                    "phrase": phrase_option,
+                    "numeric": numeric_option,
+                    "math": math_option,
+                    "oldsource": pluginname,
+                    "pyload": "1",
+                    "source": "pyload",
+                    "version": self.__version__,
+                    "name": self.__name__,
+                    "base64": isBase64,
+                    "mouse": mouse,
+                    "file-upload-01": data,
+                    "pageurl" : pageurl,
+                    "interactive" : interactive,
+                    "action": "usercaptchaupload" })
+                time.sleep(2)
+
+                if(response != ""):
+                    break;
 
             except BadHeader, e:
                 time.sleep(3)
```

Comment:
set response to None, otherwise it will not be set at all in case of an exception